From: Felix Delattre <felix@delattre.de>
Date: Sun, 29 Aug 2021 13:24:51 +0000
Subject: Fix simple pod syntax error for OpenSeaMap
Origin: https://github.com/openstreetmap/tirex/pull/37/commits/9004168e424b22ec4b27fb731489971a771af65c
Bug: https://github.com/openstreetmap/tirex/pull/37

---
 lib/Tirex/Backend/OpenSeaMap.pm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/Tirex/Backend/OpenSeaMap.pm b/lib/Tirex/Backend/OpenSeaMap.pm
index 277b0cc..b41450a 100644
--- a/lib/Tirex/Backend/OpenSeaMap.pm
+++ b/lib/Tirex/Backend/OpenSeaMap.pm
@@ -38,6 +38,8 @@ Config parameters for the map file:
 
 =item tilesize the tile size (should be 256 * scalefactor to avoid issues)
 
+=back
+
 =head1 METHODS
 
 =head2 $backend->init()
