From: Felix Delattre <felix@delattre.de>
Date: Sun, 29 Aug 2021 13:26:33 +0000
Subject: Disconnect configuration overlap between build and debian packaging
Origin: https://github.com/openstreetmap/tirex/pull/34/commits/ae3b7495083a942b4196e435759054d77721164e
Bug: https://github.com/openstreetmap/tirex/pull/34

---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 7d45111..92dce76 100644
--- a/Makefile
+++ b/Makefile
@@ -61,8 +61,6 @@ install: build
 	install -m 644 ${INSTALLOPTS} etc/renderer/mapserver/fonts.lst         $(DESTDIR)/etc/tirex/renderer/mapserver/fonts.lst
 	install -m 755 ${INSTALLOPTS} -d                                       $(DESTDIR)/etc/tirex/renderer/mapnik
 	install -m 644 ${INSTALLOPTS} etc/renderer/mapnik.conf.dist            $(DESTDIR)/etc/tirex/renderer/mapnik.conf
-	install -m 755 ${INSTALLOPTS} -d                                       $(DESTDIR)/etc/logrotate.d
-	install -m 644 ${INSTALLOPTS} debian/logrotate.d-tirex-master          $(DESTDIR)/etc/logrotate.d/tirex-master
 	install -m 755 ${INSTALLOPTS} -d                                       $(DESTDIR)/usr/share/man/man1/
 	for program in bin/*; do \
         if grep -q "=head" $$program; then \
