From: Felix Delattre <felix@delattre.de>
Date: Fri, 3 Sep 2021 18:04:05 +0000
Subject: Add debug information to tirex-backend-manager
Origin: https://github.com/openstreetmap/tirex/pull/39/commits/d7f0ae48e5fe12e6877ac8bfc0ec2feb00ed45b3
Bug: https://github.com/openstreetmap/tirex/pull/39

---
 backend-mapnik/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend-mapnik/Makefile b/backend-mapnik/Makefile
index 832358e..977a48e 100644
--- a/backend-mapnik/Makefile
+++ b/backend-mapnik/Makefile
@@ -1,6 +1,6 @@
 INSTALLOPTS=-g root -o root
 CFLAGS += -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
-CXXFLAGS = $(CFLAGS) `mapnik-config --cflags`
+CXXFLAGS = `mapnik-config --cflags` $(CFLAGS)
 CXXFLAGS += -Wall -Wextra -pedantic -Wredundant-decls -Wdisabled-optimization -Wctor-dtor-privacy -Wnon-virtual-dtor -Woverloaded-virtual -Wsign-promo -Wold-style-cast
 LDFLAGS= `mapnik-config --libs --ldflags --dep-libs`
