Description: Use pkg-config for Mapnik 4.0.0.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://github.com/openstreetmap/tirex/issues/60

--- a/backend-mapnik/Makefile
+++ b/backend-mapnik/Makefile
@@ -1,8 +1,7 @@
 INSTALLOPTS=-g root -o root
-CFLAGS += -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
-CXXFLAGS = `mapnik-config --cflags` $(CFLAGS)
+CXXFLAGS += `pkg-config --cflags libmapnik` -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
 CXXFLAGS += -Wall -Wextra -pedantic -Wredundant-decls -Wdisabled-optimization -Wctor-dtor-privacy -Wnon-virtual-dtor -Woverloaded-virtual -Wsign-promo -Wold-style-cast
-LDFLAGS= `mapnik-config --libs --ldflags --dep-libs`
+LDFLAGS += `pkg-config --libs libmapnik` -lboost_filesystem
 
 backend-mapnik: renderd.o metatilehandler.o networklistener.o networkmessage.o networkrequest.o networkresponse.o debuggable.o requesthandler.o
 	$(CXX) -o $@ $^ $(LDFLAGS)
--- a/backend-mapnik/metatilehandler.cc
+++ b/backend-mapnik/metatilehandler.cc
@@ -25,7 +25,7 @@
 #include <mapnik/agg_renderer.hpp>
 #include <mapnik/image_util.hpp>
 #include <mapnik/load_map.hpp>
-#include <mapnik/box2d.hpp>
+#include <mapnik/geometry/box2d.hpp>
 
 #if MAPNIK_VERSION >= 300000
 # include <mapnik/datasource.hpp>
